wpengine/faustjs
# Bug: Force cli package to use @wordpress/scripts

## Tasks


## Description

* This PR forces the cli `blockset` command to use the `@wordpress/scripts` package as part of the build process avoiding any potential confusion.

## Related Issue(s):



## Testing



* clean up your node_modules package
* Install the project depedencies again
* run `npm run blockset -w examples/next/block-support` it should use the `--package=@wordpress/scripts` in the log files

```
npm verb argv "exec" "wp-scripts" "start" "--package" "@wordpress/scripts" "--loglevel" "verbose" "--" "--no-watch" "--webpack-src-dir=wp-blocks" "--output-path=/Users/theo.despoudis/Workspace/faustjs/examples/next/block-support/.faust/build"
```

## Screenshots



## Documentation Changes



## Dependant PRs
